package.json

```diff
@@ -21,7 +21,12 @@
   "devDependencies": {
     "depcheck": "^0.6.4",
     "eslint": "^3.5.0",
-    "fixpack": "^2.3.1"
+    "fixpack": "^2.3.1",
+    "npm-run-all": "^3.1.0",
+    "nsp": "^2.6.1"
+  },
+  "engines": {
+    "node" : ">=6"
```

Comment:
Added your `engines` thing here. So in theory it should warn you if you try installing on Node 5 or lower.